src/main/java/com/blogify/blogapi/repository/CommentRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.blogify.blogapi.repository;
+
+import com.blogify.blogapi.model.Comment;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface CommentRepository extends JpaRepository<Comment, Long> {
```

Comment:
, String> NOT , Long>

Reply:
because the type of id in Comment is String